source/base/Profiler.ooc

```diff
@@ -67,4 +60,13 @@ Profiler: class {
 		This _profilers free()
 		This _profilers = null
 	}
+	_logResults: static func (logMethod: Func (String)) {
+		for (i in 0 .. This _profilers count) {
```

Comment:
I needed to use a loop because I got segmentation fault when trying to free the closure here.